tests/compose/test.py

```diff
@@ -88,7 +88,7 @@ def hooks():
 sys.stdout.flush()
 deadline=datetime.datetime.now()+datetime.timedelta(minutes=timeout)
 deadline=calendar.timegm(deadline.timetuple())
-print(subprocess.check_output("docker-compose -f " + compose_file + " up -d", shell=True).decode())
+print(subprocess.check_output("docker compose -f " + compose_file + " up -d", shell=True).decode())
```

Comment:
I think we should start using f-strings here. 

```suggestion
print(subprocess.check_output(f"docker compose -f {compose_file} up -d", shell=True).decode())
```